tests/testing_meshpy.py

```diff
@@ -119,14 +120,15 @@ def create_test_mesh(mesh):
     mesh.add(beam)
 
     # Add a beam line with three elements
-    create_beam_mesh_line(
+    beamset = create_beam_mesh_line(
         mesh,
         Beam3rHerm2Line3,
         material,
         [100 * random.uniform(-1, 1) for _i in range(3)],
         [100 * random.uniform(-1, 1) for _i in range(3)],
         n_el=3,
     )
+    return beamset
```

Comment:
Why do we need to change the signature of this function?

Reply:
Because I forgot to delete it - is removed.